components/job-orchestration/job_orchestration/executor/compress/fs_compression_task.py

```diff
@@ -291,15 +337,28 @@ def compress(
     paths_to_compress_json: str,
     clp_metadata_db_connection_config,
 ):
-    clp_home_str = os.getenv("CLP_HOME")
-    data_dir_str = os.getenv("CLP_DATA_DIR")
-    archive_output_dir_str = os.getenv("CLP_ARCHIVE_OUTPUT_DIR")
-    logs_dir_str = os.getenv("CLP_LOGS_DIR")
+    clp_home = pathlib.Path(os.getenv("CLP_HOME"))
```

Comment:
Not mandatory, but since we're changing the env vars, it might be a good time to define constants for these env var names.